modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -4,193 +4,57 @@ This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualB
 
-Once the installation is complete, you have to remove the live ISO after shutting down (`sudo shutdown -h now`) the VM. Once the ISO is removed, you can start the VM anew. The actual first boot process begins now. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
+=== Importing the OVA
 
+You can set up a VirtualBox virtual machine through the GUI or via the https://www.virtualbox.org/manual/UserManual.html#vboxmanage[`vboxmanage` CLI]. Below is a shell script that does most of what the graphical wizard will do. Note that this is tailored to a ova import. 
 [source, bash]
 ----
-vboxmanage storageattach "${VM_NAME}" --storagectl IDE --type dvddrive --port 1 --device 0 --medium emptydrive
-vboxmanage startvm "${VM_NAME}" --type headless
-# Once the VM is available, you can log in:
-ssh core@localhost -p 2222
-----
-
-== Installing from raw bare metal image
+OVA_PATH="SetThisWithYourPathToOVA"
 
-WARNING: The steps below are provided for advanced users, may unexpectedly break in various places with newer Fedora CoreOS images, and require a Unix environment to work. If you encounter issues consider using the xref:_installing_from_live_iso[live ISO instructions] instead.
-
-To install FCOS into a VirtualBox virtual machine using the raw disk image, follow these steps:
-
-- Download the latest raw image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
-[source, bash]
-----
-podman run --security-opt label=disable --pull=always --rm -v .:/data -w /data \
-    quay.io/coreos/coreos-installer:release download -f raw
+VBoxManage import ${OVA_PATH}
 ----
-
-Before the installation can begin you have to convert the raw disk image to a VirtualBox compatible one:
-[source,bash,subs="attributes"]
-----
-FCOS_DISK=$(realpath ~/Downloads/fedora-coreos-{stable-version}-metal.x86_64.raw.xz)
-UNPACKED_DISK=$(realpath ~/Downloads/fedora-coreos-{stable-version}-metal.x86_64.raw)
-
-# Unpack the raw disk image
-unxz "$\{FCOS_DISK}"
-----
-
 [NOTE]
 ====
-If you want to supply the Ignition configuration file URL statically, you can modify the raw disk before converting it to a VirtualBox disk image.
-
-To correctly mount the boot partition you have to find out the start point of it and the disk's sector size. Multiply the two values to calculate the mount `offset`.
-[source, bash]
-----
-TMP_FCOS_BOOT_VOLUME=$(realpath /tmp/fcos_boot_volume)
-mkdir "${TMP_FCOS_BOOT_VOLUME}"
-
-fdisk -l "${UNPACKED_DISK}"
-sudo mount -o loop,offset="${OFFSET}" "${UNPACKED_DISK}" "${TMP_FCOS_BOOT_VOLUME}"
-----
+Once imported make sure that the network adapter is set to the type you want. More then likely it will be set to NAT.
+This seems to be due to the networking settings falling back to default. 
```

Comment:
This isn't super-clear.  Let's say 1) the default network attachment is NAT, 2) you might want e.g. bridged networking so the VM is accessible on the local network, and 3) here's how to change it (is there a `vboxmanage` subcommand?).

The last sentence is more relevant to a tracker ticket than user-facing docs.